bib/cmd/bootc-image-builder/main.go

```diff
@@ -125,47 +112,79 @@ func getContainerSize(imgref string) (uint64, error) {
 	return size, nil
 }
 
-func makeManifest(c *ManifestConfig, cacheRoot string) (manifest.OSBuildManifest, error) {
-	if c.Local {
-		if err := setup.ValidateHasContainerStorageMounted(); err != nil {
-			return nil, fmt.Errorf("local storage not working, did you forget -v /var/lib/containers/storage:/var/lib/containers/storage? (%w)", err)
+// removeDuplicateRepos removes duplicities from the list of repositories
+//
+// The equality of repositories is determined by the unique Id field.
+func removeDuplicateRepos(repos []rpmmd.RepoConfig) []rpmmd.RepoConfig {
+	ids := make(map[string]bool)
+	dedupedRepos := []rpmmd.RepoConfig{}
+	for _, repo := range repos {
+		if _, value := ids[repo.Id]; !value {
```

Comment:
(nitpick) given that go will return the `zero` value for non-existing map keys this could be slightly simplified (but doesn't have to):
```go
--- a/bib/cmd/bootc-image-builder/main.go
+++ b/bib/cmd/bootc-image-builder/main.go
@@ -117,9 +117,9 @@ func getContainerSize(imgref string) (uint64, error) {
 // The equality of repositories is determined by the unique Id field.
 func removeDuplicateRepos(repos []rpmmd.RepoConfig) []rpmmd.RepoConfig {
        ids := make(map[string]bool)
-       dedupedRepos := []rpmmd.RepoConfig{}
+       dedupedRepos := make([]rpmmd.RepoConfig, 0, len(repos))
        for _, repo := range repos {
-               if _, value := ids[repo.Id]; !value {
+               if !ids[repo.Id] {
                        ids[repo.Id] = true
                        dedupedRepos = append(dedupedRepos, repo)
                }
```

Reply:
This function was removed, `images` now does everything for us.